BigQuery/tests/System/data/table-data.json

```
@@ -1,3 +1,3 @@
-{"Name":"Bilbo","Age":"111","Weight":67.2,"IsMagic":false,"Spells":[],"ImportantDates":{"TeaTime":"10:00:00","NextVacation":"2017-09-22","FavoriteTime":"2031-04-01T05:09:27"},"FavoriteNumbers":[1],"BiggerNumbers":[8],"Location":"POINT(-2.75 53.85)"}
+{"Name":"Bilbo","Age":"111","Weight":67.2,"IsMagic":false,"Spells":[],"ImportantDates":{"TeaTime":"10:00:00","NextVacation":"2017-09-22","FavoriteTime":"2031-04-01T05:09:27"},"FavoriteNumbers":[1],"BiggerNumbers":[8],"Location":"POINT(-2.75 53.85)","Json":{"HouseNumber":1234,"City":"Bangalore"}}
```

A better name to distinguish from the field name itself?
```suggestion
{"Name":"Bilbo","Age":"111","Weight":67.2,"IsMagic":false,"Spells":[],"ImportantDates":{"TeaTime":"10:00:00","NextVacation":"2017-09-22","FavoriteTime":"2031-04-01T05:09:27"},"FavoriteNumbers":[1],"BiggerNumbers":[8],"Location":"POINT(-2.75 53.85)","AddressJson":{"HouseNumber":1234,"City":"Bangalore"}}
```

---

Done